source/css/_schemes/Pisces/_sidebar.styl

```diff
@@ -100,10 +100,6 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
-
-  span.exturl {
-    font-size: 13px;
```

Comment:
Screens: before and after?

Reply:
No changes are made. Duplicate definition.

Reply:
Duplicate definition from where?

Reply:
font-size will inherit from its parent node. The screenshot already shows that nothing has changed with default setting